cookbooks/convection-box/tutorial-onset-of-convection/README.md

```diff
@@ -0,0 +1,86 @@
+Author: Lorraine J. Hwang, Ian Rose, Juliane Dannberg and the ASPECT development community
+
+# Introduction to ASPECT
+
+This notebook is based on tutorials by J. Dannberg that provide a basic introduction to ASPECT.
+This notebook demonstrates the onset of convection and the Nusselt-Rayleigh number relationship.
+
+To run, copy the contents of this directory to your workspace.
+
+
+## Running using ASPECT Jupyter Notebooks tool
+
+The current version is verified to run within the ASPECT Jupyter Notebooks tool which can be launched from the CIG website:
+
+https://geodynamics.org/resources/aspectnotebook
```

Comment:
This is not documented here or on the page linked here: It seems that you have to add $ASPECT_DIR to the path manually before you can start "aspect". @gassmoeller is that true or am I missing something?

Reply:
I believe this is in the default path for the notebook container.

Reply:
@tjhei: Lorraine is correct, $ASPECT_DIR is already set inside the tool. No need for the user to set it.

Reply:
If I click on https://geodynamics.org/resources/aspectnotebook, then 'launch tool', "new terminal" I get:
```
timoheister@geodynamics_1099_6:~$ aspect
bash: aspect: command not found
```
while ``$ASPECT_DIR/bin/aspect`` works.